runtime/gc_vlhgc/AllocationContextBalanced.cpp

```diff
@@ -689,40 +689,42 @@ MM_AllocationContextBalanced::acquireFreeRegionFromHeap(MM_EnvironmentBase *env)
 }
 
 MM_HeapRegionDescriptorVLHGC *
-MM_AllocationContextBalanced::acquireMPBPRegionFromNode(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocationContextTarok *requestingContext)
+MM_AllocationContextBalanced::acquireMPAOLRegionFromNode(MM_EnvironmentBase *env, MM_MemorySubSpace *subSpace, MM_AllocationContextTarok *requestingContext)
 {
+//	Trc_MM_AllocationContextBalanced_acquireMPAOLRegionFromNode_Entry(env->getLanguageVMThread(), this, requestingContext);
```

Comment:
did you introduce this trace point replacement? if not, remove this (I guess, we can live with BP in trace point name)

Reply:
yes, I did append new Trc_MM_AllocationContextBalanced_acquireMPAOLRegionFromNode_Entry in j9mm.tdf.